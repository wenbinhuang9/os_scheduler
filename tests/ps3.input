3 2 5 20
1 0 10 100
5 5 5 80