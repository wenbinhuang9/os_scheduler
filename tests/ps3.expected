1 10 0 10
3 15 8 13
5 15 5 10
4 11